tutorials/java-server/_posts/2017-05-28-servlets.md

```diff
@@ -52,8 +52,8 @@ Just like any other Java file, you have to compile your servlet into a `.class`
 
 Read [Anatomy of a Web App](/tutorials/java-server/web-app) for a few approaches you can use to compile your server classes. To help you get started, here are some example projects:
 
-- You can see an example project that uses the command line to package a web app [here](https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/examples/java-server/java-server-example-projects/hello-world-command-line), and you can download it as a `.zip` file [here](https://downgit.github.io/#/home?url=https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/examples/java-server/java-server-example-projects/hello-world-command-line).
-- You can see an example project that uses Maven to package a web app [here](https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/examples/java-server/java-server-example-projects/hello-world-maven), and you can download it as a `.zip` file [here](https://downgit.github.io/#/home?url=https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/examples/java-server/java-server-example-projects/hello-world-maven).
+- You can see an [hello world](https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/examples/java-server/java-server-example-projects/hello-world-command-line) example project that uses the command line to package a web app, and you can download it as a `.zip` file from the [DownGit link](https://downgit.github.io/#/home?url=https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/examples/java-server/java-server-example-projects/hello-world-command-line).
```

Comment:
This should probably be:

```
You can see a [hello world example project]((https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/examples/java-server/java-server-example-projects/hello-world-command-line) that uses the command line to package a web app...
```

Specifically, I think it should be **a** instead of **an** to make sure it's still grammatically correct.

(Same thing for the next one as well.)